### PR TITLE
Add a page about terminology and add products introduction to Getting…

### PR DESCRIPTION
**Pull request by rebekaburnett**

### CODE REVIEW

File: vipps-terminology.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+* BankID - Software that certifies the identity of a person.
+
+* D-number - A Norwegian identification number given to foreign persons who do not have a Norwegian national identity number, but have a need for identification for the Norwegian authorities.
+
+* Deeplink - The Vipps "deeplink" opens the payment page in Vipps where the user
+accepts a payment. This is an integrated part of the Vipps payment process,
+and the link should never be sent in an SMS or email.
+
+* Direct capture - **Direct capture** causes all payment reservations to be instantly be captured.
```
Comment (Haldormyre) [inline vipps-terminology.md:25]:
```suggestion
* Direct capture - **Direct capture** causes all payment reservations to be instantly captured.
```

File: vipps-terminology.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+This is intended for situations where the product or service is immediately
```
Comment (Haldormyre) [inline vipps-terminology.md:26]:
```suggestion
This is intended for situations where the product or service is immediate
```

File: vipps-terminology.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+provided to the customer, e.g. digital services.
+
+* Idempotency-Key - An idempotency header ensures that, if the merchant retries a request with the same idempotency key, the retried request will not make additional changes.
+
+* Merchant - A sale unit that integrates with Vipps for online commerce.
+
+* Merchant Serial Number (MSN) - A unique id given to a sale unit by Vipps. This is used to identify a merchant in API logs.
```
Comment (Haldormyre) [inline vipps-terminology.md:33]: It is used for a bit more, e.g the point below

File: vipps-terminology.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+* [*Finanstilsynet*](https://www.finanstilsynet.no) - A Norwegian agency that supervises companies
+
+* *Faktura* - invoice
+
+* *Fødselsnummer* - Norwegian identity number
+
+* *Månedsabonnement* - Monthly subscription
+
+* *Neste* - Next
```
Comment (cloveras) [inline vipps-terminology.md:62]: ?

File: vipps-terminology.md (hunk 1 above)
Comment (Haldormyre) [reply vipps-terminology.md:25]: Or other way around i suppose

File: vipps-getting-started.md [hunk 5]
```diff
@@ -57,6 +60,27 @@ Document version: 3.3.4.
 
 <!-- END_TOC -->
 
+## Product types
```
Comment (rebekaburnett) [inline vipps-getting-started.md:63]: What do you think about these descriptions?
I put them in Getting Started to help people get oriented, but they could also go on the terminology page.

File: vipps-terminology.md (hunk 3 above)
Comment (rebekaburnett) [reply vipps-terminology.md]: @Haldormyre Please have a look now :-)

Comment (Haldormyre) [reply vipps-terminology.md]: That seems great!

**Pull request merged**